README.md

```diff
@@ -59,10 +60,11 @@ See the file [COPYING](COPYING) for more details.
 
 HISTORY
 -------
+XQF was originally written by Roman Pozlevich from Pereslavl-Zalessky, Russia. Over 17 years of existence the project has had the following major developers:
```

Comment:
@skybon:
> Specifying homeplace of each developer is important if our names are more than just 'author' line in git log :). Appending (like Ludwig Nussel, town_name, Germany) should be enough.

I personally changed my homeplace 5 times in the last 6 years, and it could be the same for all of us. I think names are sufficient. Since the project is 17 years old, I can't imagine where I will live in the next 17 years and if I will live. Our homeplace bring nothing to XQF, but “who we are” changes how XQF is made. Internationalization is already visible in the diversity of names, so I think names are sufficient.

Our names are already more than an “author” line in the git log: [AUTHORS.md](https://github.com/XQF/xqf/blob/master/AUTHORS.md). I wrote this file picking all names from git log and git comments and ChangeLog (for example statements like “patch from someone” in commit done by other one, yes I've read all the past comments of the git history and all the ChangeLog).

@zturtleman:

> Saying "XQF is an active project." in the history section seems out of place to me. I'd either put it in the initial description or (more likely) leave it out. As a user I wouldn't trust that statement to be updated when the project became inactive and look at the commits history to find out with or without the statement.

You are right, an inactive project can have an unupdated “this project is active” statement, which means nothing. The sentence you propose is very good, it says there is many people, it says the project is old, and it says the project is long-lasting (so, old does not mean inactive), this is what I wanted to say.

> XQF was originally written by Roman Pozlevich in 1998. It has been maintained and improved by a devoted team over the years with following major developers:

:+1: